src/utils/config.ts

```diff
@@ -94,7 +99,7 @@ export async function findTargetInConfiguration(
       : undefined
 
     if (fallBackTargetJson) {
-      process.logger?.warn(
+      process.logger?.info(
         `Target ${targetName || ''} was not found. Falling back to target ${
```

Comment:
`targetName` will always be an empty string here

Reply:
got it, fixed.